broker-daemon/relayer/market-watcher.js

```diff
@@ -26,6 +27,17 @@ class MarketWatcher extends EventEmitter {
     this.RESPONSE_TYPES = RESPONSE_TYPES
 
     this.setupListeners()
+    this.createChecksum()
```

Comment:
I think you should set `this.checksum = new Checksum()` here instead of in createChecksum. As it is right now, it's a bit confusing to set an instance variable in an obscure function instead of here.

You could have a function that populates the checksum that you call after instantiating it here.

Similar thoughts for this.migratating and this.migrate()